src/extends/KV/index.tsx

```diff
@@ -216,13 +260,27 @@ class KV extends Component<Props, State> {
                   </If>
                 </Form.Item>
               </Col>
+              <Col span={5}>
+                <Form.Item>
+                  <Select
+                    showSearch={true}
+                    dataSource={options}
+                    {...init(`envType-${item.key}`)}
+                    label={'Type'}
+                    placeholder={i18n.t('Please select')}
+                    locale={locale().Select}
+                    onSearch={this.onSearch}
```

Comment:
Did not handle the value change?

Reply:
Like this ?

Reply:
<img width="452" alt="1667461036406" src="https://user-images.githubusercontent.com/105778708/199668553-9cf4d703-5d78-4858-b5ee-86bffc530d18.png">
If the type changes, the corresponding formItem will change too. Or this means something else ？

Reply:
It's great! Let's make the style better.